src/routes/(iframed)/settings/+page.svelte

```diff
@@ -7,7 +7,9 @@
     { value: 18, text: "Fast" }
   ];
 
-  function navigateBack() {
+  function navigateBack(e: MouseEvent) {
+    e.preventDefault();
```

Comment:
Without this when I left settings it seemed to actually hit the `href` which didn't have the game query param

Adding this allowed me to go back and forth between the settings page and the game page